### DISCUSSION TITLE
Request: Make albums be the default view when clicking on an artist (or make it a setting)

### DISCUSSION THREAD

**Discussion by fernplant** [music-assistant/support#1054] [Feature requests and ideas]
Small request but hopefully others would benefit: make list of albums the default view when navigating by artist (ie clicking Artists then an artist's name then displaying list of their albums instead of list of tracks).
I like to listen to full albums and this is how I navigate, pretty annoying to always have to click to albums and also annoying that it doesnt remember I was on the album view when clicking back, it goes back to track view by default

**Comment by OzGav** [comment]
Said the same thing here https://github.com/music-assistant/hass-music-assistant/issues/748#issuecomment-1197966559 so I have upvoted you!

Also I proposed some sort changes here https://github.com/music-assistant/hass-music-assistant/discussions/523

**Reply by fernplant** [reply]
Oh nice, Im not alone then :)
